src/network/packets.h

```diff
@@ -67,4 +67,8 @@
 #define PACKET_INSPECTION_DATATYPE_INT 1
 #define PACKET_INSPECTION_DATATYPE_FLOAT 2
 
+enum class PacketErrorCode : uint8_t {
```

Comment:
That seems not to be defined till now. On BNO it sends the following.
<https://github.com/SlimeVR/SlimeVR-Tracker-ESP/blob/main/src/sensors/bno080sensor.cpp#L220-L223>
//Get the reason for the last reset
//1 = POR, 2 = Internal reset, 3 = Watchdog, 4 = External reset, 5 = Other